doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -178,53 +160,45 @@ some incompatibilities.
    equality must contain all the quantified variables in order for congruence to
    match against it.
 
-.. example::
-
-   .. coqtop:: reset all
-
-      Theorem T (A:Type) (f:A -> A) (g: A -> A -> A) a b: a=(f a) -> (g b (f a))=(f (f a)) -> (g a b)=(f (g b a)) -> (g a b)=a.
-      intros.
-      congruence.
-      Qed.
-
-      Theorem inj (A:Type) (f:A -> A * A) (a c d: A) : f = pair a -> Some (f c) = Some (f d) -> c=d.
-      intros.
-      congruence.
-      Qed.
+   Increasing the maximum number of hypotheses may solve
+   problems that would have failed with a smaller value.  It will make failures slower but it
+   won't make successes found with the smaller value any slower.
+   You may want to use :tacn:`assert` to add some lemmas as
+   hypotheses so that :tacn:`congruence` can use them.
 
-.. tacv:: congruence @natural
+   .. example::
 
-  Tries to add at most :token:`natural` instances of hypotheses stating quantified equalities
-  to the problem in order to solve it. A bigger value of :token:`natural` does not make
-  success slower, only failure. You might consider adding some lemmas as
-  hypotheses using assert in order for :tacn:`congruence` to use them.
+      .. coqtop:: reset all
 
-.. tacv:: congruence with {+ @term}
-   :name: congruence with
+         Theorem T (A:Type) (f:A -> A) (g: A -> A -> A) a b: a=(f a) -> (g b (f a))=(f (f a)) -> (g a b)=(f (g b a)) -> (g a b)=a.
+         intros.
+         congruence.
+         Qed.
 
-   Adds :n:`{+ @term}` to the pool of terms used by :tacn:`congruence`. This helps
-   in case you have partially applied constructors in your goal.
+         Theorem inj (A:Type) (f:A -> A * A) (a c d: A) : f = pair a -> Some (f c) = Some (f d) -> c=d.
+         intros.
+         congruence.
+         Qed.
 
-.. exn:: I don’t know how to handle dependent equality.
+   .. exn:: I don’t know how to handle dependent equality.
 
   The decision procedure managed to find a proof of the goal or of a
   discriminable equality but this proof could not be built in Coq because of
   dependently-typed functions.
```

Comment:
Missing indentation makes the build fail:

```suggestion
     The decision procedure managed to find a proof of the goal or of a
     discriminable equality but this proof could not be built in Coq because of
     dependently-typed functions.
```